Sources/SmithyRetriesAPI/RetryStrategyOptions.swift

```diff
@@ -50,7 +50,7 @@ public struct RetryStrategyOptions {
     ///   - availableCapacity: The number of available tokens in a retry quota.  Defaults to 500.
     ///   - maxCapacity: The max number of tokens in a retry quota.  Defaults to 500.
     public init(
-        backoffStrategy: RetryBackoffStrategy = ExponentialBackoffStrategy(),
+        backoffStrategy: RetryBackoffStrategy,
```

Comment:
`ExponentialBackoffStrategy` was moved off to the implementation module, so this default value can't be set anymore - backoff strategy must be passed in when `RetryStrategyOptions` is created.